configure.py

```diff
@@ -772,11 +772,11 @@
          'memory footprint, but also implies no just-in-time compilation ' +
          'support, thus much slower execution)')
 
-parser.add_argument('--v8-enable-object-print',
+parser.add_argument('--v8-disable-object-print',
```

Comment:
We could keep `--v8-enable-object-print` as a valid option in addition to adding `--v8-disable-object-print` in case anyone is using it, otherwise they'll break

Reply:
Thanks for your suggestion, I keep the `-v8-enable-object-print` flag and add the `--v8-disable-object-print` flag.
If I specify both flags at the same time, We get an exception with a message like `Only one of the --v8-enable-object-print or --v8-disable-object-print options an be specified at a time.`.

Reply:
Downloading the source tarball on Nodejs' website for LTS 18.13.0 yields:
```
python3 ./configure --prefix=/opt/base --shared-zlib --v8-disable-object-print --without-corepack --without-dtrace --without-etw --without-inspector --without-intl --without-node-code-cache --without-node-snapshot --without-npm --without-ssl
Node.js configure: Found Python 3.9.10...
Traceback (most recent call last):
  File "/Users/Hames/Repos/core/contrib/node-v18.13.0/./configure", line 28, in <module>
    import configure
  File "/Users/Hames/Repos/core/contrib/node-v18.13.0/configure.py", line 2062, in <module>
    configure_v8(output)
  File "/Users/Hames/Repos/core/contrib/node-v18.13.0/configure.py", line 1547, in configure_v8
    raise Exception(
Exception: Only one of the --v8-enable-object-print or --v8-disable-object-print options can be specified at a time.
```

Even though the `enable` flag is never specified.

This is because the `options` object has the argparse defaults resolved, which means the enable flag is `True` if not specified... :/